src/msw/textctrl.cpp

```diff
@@ -2718,14 +2725,53 @@ wxMenu *wxTextCtrl::MSWCreateContextMenu()
     return m;
 }
 
+namespace
+{
+
+inline void ApplyZoomFactor(HWND hwnd, float scale)
+{
+    UINT num = 1;
+    UINT denom = 1;
```

Comment:
It looks like we could just define them as `[WL]PARAM` and avoid some casts.

Reply:
`EM_GETZOOM` uses them by reference, I don't think that will work.